Library/Homebrew/tap.rb

```diff
@@ -62,6 +62,7 @@ def initialize(user, repo)
     @repo = repo
     @name = "#{@user}/#{@repo}".downcase
     @path = TAP_DIRECTORY/"#{@user}/homebrew-#{@repo}".downcase
+    @remote = remote
```

Comment:
otherwise get nil when doing the compare below `raise TapRemoteMismatchError.new(name, @remote, remote) unless @remote == remote`